### PR TITLE
Add 'Submission and Access' rules and cleanups

### PR DESCRIPTION
**Pull request by jonathanirvings**

### CODE REVIEW

File: rules.md [hunk 1]
```diff
@@ -83,3 +83,13 @@ Delegation Leaders have the responsibility of ensuring that all members of their
    * “ANSWERED IN TASK DESCRIPTION (EXPLICITLY OR IMPLICITLY)” – The task description contains sufficient information. The contestant should read it again carefully.
    * “INVALID QUESTION” – The question is most likely not phrased so that a yes/no answer would be meaningful. The contestant is encouraged to rephrase the question.
    * “NO COMMENT” – The contestant is asking for information that the Scientific Committee cannot give.
+
+<br>
+
+### F. Submission and Access
```
Comment (fushar) [inline rules.md:89]: Maybe add something like this as the last point:

- For the sake of respecting fairness of this newly-introduced online contest format, each contestant is expected to show a good sportsmanship by following the above rules honestly.

File: rules.md [hunk 2]
```diff
@@ -83,3 +83,13 @@ Delegation Leaders have the responsibility of ensuring that all members of their
...
+
+* All submission from a contestant must be written by the contestant.
+* The contestants are allowed to use any code written before the competition, as long as the code is written by the contestant.
```
[the remark this answers is not in the document]
Comment (jonathanirvings) [reply rules.md]: Done.

File: index.html [hunk 3]
```diff
@@ -4,6 +4,14 @@
 ---
 
 <div>
+  <div class="pt-card card__title">
+    <h3 class="card__title__text">Updated rules</h3>
+    <p class="card__title__right">24 May 2020</p>
+  </div>
+  <div class="pt-card card__content">
+    <p>
+      A new section "Submission and Access" is added to the competition rules. Since it is not mandatory for participants to take the contest at an official contest site, APIO 2020 committee has also modified what resources can the contestants use, submit, and access during the competition.
```
[the remark this answers is not in the document]
Comment (jonathanirvings) [reply index.html]: Done.

File: rules.md [hunk 4]
```diff
@@ -83,3 +83,13 @@ Delegation Leaders have the responsibility of ensuring that all members of their
    * “ANSWERED IN TASK DESCRIPTION (EXPLICITLY OR IMPLICITLY)” – The task description contains sufficient information. The contestant should read it again carefully.
    * “INVALID QUESTION” – The question is most likely not phrased so that a yes/no answer would be meaningful. The contestant is encouraged to rephrase the question.
    * “NO COMMENT” – The contestant is asking for information that the Scientific Committee cannot give.
+
+<br>
+
+### F. Submission and Access
+
+* All submission from a contestant must be written by the contestant.
```
[the remark this answers is not in the document]
Comment (jonathanirvings) [reply rules.md]: Done.

File: rules.md [hunk 5]
```diff
@@ -83,3 +83,13 @@ Delegation Leaders have the responsibility of ensuring that all members of their
...
+* The contestants are allowed to use any code written before the competition, as long as the code is written by the contestant.
+* The contestants are prohibited to talk or discuss anything related to the content of the competition with anyone, including other contestants, except to the committee of the competition using the clarification request explained in the earlier section.
```
[the remark this answers is not in the document]
Comment (jonathanirvings) [reply rules.md]: Done.

File: rules.md [hunk 6]
```diff
@@ -83,3 +83,13 @@ Delegation Leaders have the responsibility of ensuring that all members of their
...
+* The contestants are allowed to use any sources of information on the Internet, as long as the contestant does not violate the previous rule. In particular, it is forbidden to copy-paste somebody else's code on the internet, or posting anything related to the content of the competition in an online forum.
```
[the remark this answers is not in the document]
Comment (jonathanirvings) [reply rules.md]: Done.

File: rules.md (hunk 1 above)
Comment (jonathanirvings) [reply rules.md:89]: Done.

**Pull request merged**